addon/components/sidenotes-wrapper.js

```diff
@@ -20,7 +20,16 @@ export default class SidenotesWrapperComponent extends Component {
   }
 
   get key() {
-    return this.args.key ?? '@identity';
+    return this.args.key ? 'key' : '@identity';
+  }
+
+  get itemsPrepared() {
+    return this.args.items.map((item) => {
+      if (this.args.key) {
+        item['key'] = `${item[this.args.key]}-${item.y}`;
```

Comment:
```suggestion
        item['_key'] = `${item[this.args.key]}-${item.y}`;
```
To avoid conflict with existing properties

Reply:
https://github.com/concordnow/ember-sidenotes/pull/76/commits/56279d81dbac374ee3e395fb9f2049797ee6421e